pages/linux/lid.md

```diff
@@ -0,0 +1,10 @@
+# lid
+
+> NOTE: This page is currently a redirection stub. If you are familiar with this program, please open a pull request!
```

Comment:
```suggestion
> NOTE: This page is currently a redirection stub. If you are familiar with this program, please open a pull request.
```
This should fix `tldr-bot`'s error.

Reply:
I think there is a case to be made for an exception here. If it's up to me, I would like to dismiss this CI error and merge anyway.

Reply:
Okay sure. We're pretty much bikeshedding at this point.

Reply:
Oh okay, I did not know that. I thought it was just an advisory sort of thing.